gamedev-iitk/tower-defense
# Github Actions to create a card for new issues

**Description:** Connecting Github and Trello

**Steps to test:**
1.Create a trial issue on github
2. You will observe a card in 'Inbox' list in trello
